critical/lambda_logging.tf

```diff
@@ -74,6 +74,14 @@ resource "aws_ssm_parameter" "log_destination_arn_experience" {
   value = module.kinesis_log_destination.cloudwatch_log_destination.arn
 }
 
+resource "aws_ssm_parameter" "log_destination_arn_experience_cloudfront" {
```

Comment:
Chatting with @jamieparkinson, this is in here as a `plan` indicated it would be deleted and we're being cautious about making changes to applied infra here.